Sming/system/include/stringutil.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * StringUtil.h
+ *
+ *  Contains utility functions for working with char strings.
+ *
+ *  Created on: 26.01.2017
+ *  Author: (github.com/)ADiea
+ */
+
+#ifndef INCLUDE_STRINGUTIL_H_
+#define INCLUDE_STRINGUTIL_H_
+
```

Comment:
@ADiea Can you add also C linkage so that these functions are also available in C code?